CIP-XXXX/README.md

```diff
@@ -0,0 +1,114 @@
+---
+CIP: 382?
```

Comment:
```suggestion
CIP: ?
```
I prefer 382, but the overhead it adds to the process is something we (editors) are keen to reduce